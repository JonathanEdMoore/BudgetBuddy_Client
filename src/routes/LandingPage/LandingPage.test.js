import React from 'react'
import ReactDOM from 'react-dom'
import LandingPage from './LandingPage'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('LandingPage smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><LandingPage /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('LandingPage snapshot test', () => {
  const wrapper = shallow(<LandingPage />)
  expect(toJson(wrapper)).toMatchSnapshot()
})