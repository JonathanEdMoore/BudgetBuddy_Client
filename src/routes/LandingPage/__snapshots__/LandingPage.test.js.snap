// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <MainHeader />
  <section>
    <header>
      <h3>
        A 21st Century Way to Manage Money
      </h3>
    </header>
    <img
      src="Transactions.png"
    />
    <p>
      BudgetBuddy helps you manage your finances. You can enter a purchase or deposit and BudgetBuddy will update your balance for you.
    </p>
  </section>
  <section>
    <header>
      <h3>
        Record your Deposits and Withdrawals
      </h3>
    </header>
    <img
      src="CreateTransaction.png"
    />
    <p>
      The app allows you to enter data about your withdrawals and deposits. Once it receives that information, it updates your checkbook balance letting you know your current balance
    </p>
  </section>
  <section>
    <header>
      <h3>
        Keep Track of Your Finances
      </h3>
    </header>
    <p>
      The app provides you with a virtual balance book and updates after each entry. The app also allows you to set up recurring deposits and withdrawals!
    </p>
  </section>
  <section>
    <header>
      <h3>
        Start Saving Now!
      </h3>
      <Link
        to="signup"
      >
        <button>
          Sign Up
        </button>
      </Link>
    </header>
  </section>
</div>
`;
